documentation/pwa/tutorial-pwa-introduction.asciidoc

```diff
@@ -6,24 +6,25 @@ layout: page
 
 = Introduction
 
-Progressive Web Applications (PWAs) combine new technologies with established best practices. They allow you to create reliable, accessible, and engaging experiences that give users a native-like experience, with a user friendly opt-in installation flow.
+Progressive Web Applications (PWAs) combine new technologies with established best practices. They allow you to create reliable, accessible, and engaging applications that give users a native-like experience, with a user-friendly opt-in installation flow, and an ability to work offline.
```

Comment:
Maybe "and _capability_ to work offline" would be a better choice of word as the subject is a thing rather than a person.